storage/rocksdb/ha_rocksdb.cc

```diff
@@ -508,6 +508,7 @@ static std::unique_ptr<rocksdb::DBOptions> rdb_init_rocksdb_db_options(void) {
   o->listeners.push_back(std::make_shared<Rdb_event_listener>(&ddl_manager));
   o->info_log_level = rocksdb::InfoLogLevel::INFO_LEVEL;
   o->max_subcompactions = DEFAULT_SUBCOMPACTIONS;
+  o->max_open_files = -2; // auto-tune to 50% open_files_limit
```

Comment:
Magic number in the code, introduce some constexpr

Reply:
Ack and not a bad idea at all @laurynas-biveinis . The switch to '-2' over '0' was a last second change due to some speculation and guessing on rocksdb expectations. As long as the UI is 'locked in' at this point for GA is the main thing. I can/will come back and make the code more correct to fit this immediately. I have https://jira.percona.com/browse/DOC-661 opened now to ensure that this behavior is clear.

Reply:
If you agree on the string sysvar, then it depends whether the change can be made before the GA or afterwards. In one case the -2 -1 numeric options may be removed, in the other not, and it's fine both ways.

Reply:
Right now, I am resigned to leave as-is since there is not much more time for re-test and re-review with holidays and such. I _believe_ that a change such as you suggest would be invisible to the UI and would make forward/backward compatibility a non-issue. My main concern is changing the expectation or behavior of the UI over a code completeness at this stage combined with our compressed timeline. Switching to a string right now would introduce a bit more code that would need testing and more docs and more chances of breaking something. We can debate if there is any left tomorrow standup.

Reply:
Yes it's OK to change later, absolutely no reason to rush this